books/api/taxes.py

```diff
@@ -0,0 +1,193 @@
+from dataclasses import dataclass
+from saferite.core import ZohoBooksBase, strict_types
+from books.data import TaxData
+
+
+@dataclass
+class Taxes:
+    token:str
+    organization_id: str
+
+    def __post_init__(self):
+        self.base = ZohoBooksBase(self.token, self.organization_id)
+        self.module  = "settings"
+    @strict_types
+    def create( self, data:TaxData):
+        """Create a tax which can be associated with an item.
+
+        Args:
+            data (TaxData)
+        """
+        return self.base.api._standard_call(f'{self.module}/taxes', 'post', data=data)
```

Comment:
I believe we must pass data.json instead of data. Also, I'm not sure if we also have to wrap it in str.